src/python/gem5/resources/resource.py

```diff
@@ -1108,9 +1117,9 @@ def Resource(
     This function was created to maintain backwards compability for v21.1.0
     and prior releases of gem5 where `Resource` was a class.
 
-    In the interests of gem5-resource specialization, the `Resource` class
-    has been dropped. Instead users are advized to use the `obtain_resource`
-    function which will return the correct `AbstractResource` implementation.
+    In the interests of gem5-resource specialization, the ``Resource`` class
+    has been dropped. Instead users are advized to use the ``obtain_resource``
```

Comment:
```suggestion
    has been dropped. Instead users are advised to use the ``obtain_resource``
```
This is not related to the reformat but advised is misspelled as advized.